### PR TITLE
IGVF-1299-image-file

### PR DESCRIPTION
**Pull request by ian-whaling**

### CODE REVIEW

File: src/igvfd/schemas/image_file.json [hunk 1]
```diff
@@ -0,0 +1,86 @@
+{
+    "title": "Image File",
+    "$id": "/profiles/image_file.json",
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "description": "A file containing image data in jpg or png format.",
```
Comment (gabdank) [inline src/igvfd/schemas/image_file.json:5]: I think the description should not limit to specific file formats.

File: src/igvfd/schemas/image_file.json [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    "dependentSchemas": {
+        "$merge": [
+            "file.json#/dependentSchemas"
+        ]
+    },
+    "type": "object",
+    "properties": {
+        "schema_version": {
+            "default": "2"
+        },
```
Comment (gabdank) [inline src/igvfd/schemas/image_file.json:48]: why 2?

Comment (ian-whaling) [reply src/igvfd/schemas/image_file.json:48]: left over from copying another schema. I've set it back to 1.

File: src/igvfd/schemas/image_file.json (hunk 1 above)
Comment (ian-whaling) [reply src/igvfd/schemas/image_file.json]: fixed.

**Pull request merged**